CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-### main
+### 2.3.0-beta.1
 
 * Fix build errors when using Flutter SDK 3.24.
 * Add `GestureState` to `MapContentGestureContext` to indicate whether gesture has been started, its touches have changed or it has ended.
+* Bump Maps SDK to 11.7.0-beta.1
```

Comment:
Should we add the Turf major version bump to 3 here?

Reply:
I don't think it should have any effect on dart side, that's iOS-specific change.

Reply:
OK, good to leave out then ✅